common/lib/xblock/xblock_discussion/xblock_discussion.py

```diff
@@ -0,0 +1,104 @@
+# -*- coding: utf-8 -*-
+"""
+Discussion XBlock
+"""
+import json
+import logging
+
+from xblockutils.resources import ResourceLoader
+from xblockutils.studio_editable import StudioEditableXBlockMixin
+
+from xblock.core import XBlock
+from xblock.fields import Scope, String, UNIQUE_ID
+from xblock.fragment import Fragment
+
+log = logging.getLogger(__name__)
+LOADER = ResourceLoader(__name__)
```

Comment:
@pomegranited if you allow me to add my two cents, we've got into habit of allowing lowercased `loader` variable to be in line with `log` name. - maybe we should just suppress it here or add it to pylintrc.

Minor nit though, don't spend more than 10 minutes to address it (and feel free to ignore - I'm on vacation, so I should be here anyway)